### PR TITLE
BugFix: Get service entrypoint also account for front-end

### PR DESCRIPTION
**Pull request by pcrespov**
## What do these changes do?

- FIX: ``GET v0/catalog/services/simcore%2Fservices%2Ffrontend%2Ffile-picker/1.0.0`` for any front-end service failed

### CODE REVIEW

**Review by mguidon — APPROVED** [review]
Thanks

**Review by mguidon — APPROVED** [review]

**Review by odeimaiz — APPROVED** [review]

File: services/catalog/src/simcore_service_catalog/services/frontend_services.py [hunk 1]
```diff
@@ -25,11 +29,11 @@ def _file_picker_service() -> ServiceDockerData:
...
 
 
-def _node_group_service() -> ServiceDockerData:
+def create_node_group_service() -> ServiceDockerData:
     return ServiceDockerData(
-        key="simcore/services/frontend/nodes-group",
+        key=f"{FRONTEND_SERVICE_KEY_PREFIX}/nodes-group",
         version="1.0.0",
-        type="frontend",
+        type=ServiceType.FRONTEND,
```
Comment (sanderegg) [inline services/catalog/src/simcore_service_catalog/services/frontend_services.py:36]: why here frontend and not the file-picker? these should be the same.

**Review by sanderegg — APPROVED** [review]
good. only one minor.

File: services/catalog/src/simcore_service_catalog/services/frontend_services.py (hunk 1 above)
Comment (pcrespov) [reply services/catalog/src/simcore_service_catalog/services/frontend_services.py:36]: I asked the same question to @odeimaiz. For some reason the file-picker was never set as front-end. Should I set it as front-end?

Comment (odeimaiz) [reply services/catalog/src/simcore_service_catalog/services/frontend_services.py:35]: OK, I just checked the frontend code. It doesn't care what type the file-picker is

**Pull request merged**